src/mbgl/renderer/sources/render_tile_source.cpp

```diff
@@ -19,15 +21,23 @@ bool RenderTileSource::isLoaded() const {
 }
 
 void RenderTileSource::upload(gfx::UploadPass& parameters) {
-    tilePyramid.upload(parameters);
+    for (auto& tile : renderTiles) {
+        tile.upload(parameters);
+    }
 }
 
 void RenderTileSource::prepare(const SourcePrepareParameters& parameters) {
-    tilePyramid.prepare(parameters);
+    renderTiles.clear();
+    for (auto& entry : tilePyramid.getRenderTiles()) {
```

Comment:
As the tile pyramid no longer operates with render tiles, maybe worth renaming some methods and variables that still use `render` in their names.

Reply:
Do you have name candidates in mind?

Reply:
What about `getTiles()`?

Reply:
`getTiles` is already there :)  maybe `getRenderedTiles` ?